boards/arm/nucleo_h745zi_q/nucleo_h745zi_q_m4.dts

```diff
@@ -16,7 +16,7 @@
 	chosen {
 		zephyr,console = &uart8;
 		zephyr,shell-uart = &uart8;
-		zephyr,sram = &sram1;
+		zephyr,sram = &ahb_sram1;
```

Comment:
For a complete use case, I think it would make sense to do the equivalent change on m7 side.

Reply:
Please clarify.

Reply:
Something like `zephyr,sram = &axi_sram;` for m7 core of the same board.

Reply:
Done.